packages/core/parcel-bundler/src/Resolver.js

```diff
@@ -60,7 +60,17 @@ class Resolver {
     let module = await this.resolveModule(filename, parent);
```

Comment:
We may want to adjust this slightly so `resolveModule` is not called when using pnp, since that will still look for a node_modules directory (it calls `findNodeModulePath`). We'll need to split out the rest of that method so that things like aliases and builtins are still applied, but it doesn't search the filesystem.

Reply:
Alternatively, `resolveModule` could also check for pnp and skip looking for node_modules if it is available.

Reply:
Good idea, done 👍